lms/static/coffee/fixtures/calculator.html

```diff
@@ -1,29 +1,30 @@
+        <span class="icon fa fa-calculator" aria-hidden="true"></span>
+    </button>
+    
     <div id="calculator_wrapper">
-      <form id="calculator">
-        <div class="input-wrapper">
-          <input type="text" id="calculator_input" tabindex="-1" />
-          <div class="help-wrapper">
-            <a id="calculator_hint" href="#" role="button" aria-haspopup="true" tabindex="-1">Hints</a>
-            <ul id="calculator_input_help" class="help" aria-activedescendant="hint-integers" role="tooltip" aria-hidden="true">
-                <li class="hint-item" id="hint-moreinfo" tabindex="-1">
-                  <p>
-                    <span class="bold">For detailed information, see
-                      <a id="hint-link-first" href="http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/completing_assignments/SFD_mathformatting.html#math-formatting">Entering Mathematical and Scientific Expressions</a> in the
-                      <a id="hint-link-second" href="http://edx-guide-for-students.readthedocs.io/en/latest/index.html">EdX Learner's Guide</a>.
-                    </span>
-                  </p>
-                </li>
-                <li class="hint-item" id="hint-integers" tabindex="-1"><p><span class="bold">Integers:</span> 2520</p></li>
-                <li class="hint-item" id="hint-decimals" tabindex="-1"><p><span class="bold">Decimals:</span> 3.14 or .98</p></li>
-            </ul>
-          </div>
-        </div>
-        <input id="calculator_button" type="submit" title="Calculate" arial-label="Calculate" value="=" tabindex="-1" />
-        <input type="text" id="calculator_output" readonly tabindex="-1" />
-      </form>
+        <form id="calculator">
+            <div class="input-wrapper">
+                <input type="text" id="calculator_input" tabindex="-1" />
+                <div class="help-wrapper">
+                    <button type="button" id="calculator_hint" aria-haspopup="true">${_('Hints')</button>
+                    <ul id="calculator_input_help" class="help" data-calculator-hint="hint-integers" role="tooltip" aria-hidden="true">
+                        <li class="hint-item" id="hint-moreinfo" tabindex="-1">
+                            <p>
+                                <span class="bold">For detailed information, see
+                                <a id="hint-link-first" href="http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/completing_assignments/SFD_mathformatting.html#math-formatting">Entering Mathematical and Scientific Expressions</a> in the
+                                <a id="hint-link-second" href="http://edx-guide-for-students.readthedocs.io/en/latest/index.html">EdX Learner's Guide</a>.
+                                </span>
+                            </p>
+                        </li>
+                        <li class="hint-item" id="hint-integers" tabindex="-1"><p><span class="bold">${_('Integers:')</span> ${_('2520')</p></li>
```

Comment:
The lines in the main file aren't translated (don't need to be). I should update the fixture though.